Makefile

```diff
@@ -1,9 +1,12 @@
 .DEFAULT_GOAL := default
 
 lint:
-	docker-compose run --rm lint
+	docker run -it --rm -v `pwd`:/plugin:ro buildkite/plugin-tester
 
 test:
 	docker run -it --rm -v `pwd`:/plugin:ro buildkite/plugin-tester
 
-default: lint test
+shellcheck:
+	docker run -it --rm -v `pwd`:/app -w /app koalaman/shellcheck hooks/** bin/** --exclude=SC1071
```

Comment:
Ditto
```suggestion
	docker run -it --rm -v "$PWD":/app -w /app koalaman/shellcheck hooks/** bin/** --exclude=SC1071
```

Reply:
Addressed in 2f8e498f1810b0fc0058cc0e37fd619c0cb9ff23 – had to use the `${}` syntax to make it work